.changeset/shaggy-kids-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/svelte': minor
```

Comment:
Let's mark this as patch as I think it's a bug fix and for Svelte 5 support (which is still experimental), we can consider any change as patches.
```suggestion
'@astrojs/svelte': patch
```